dashboard/app/controllers/api/v1/schools_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::SchoolsController < ApplicationController
+  # GET /api/v1/school/<school_district_id>/<school_type>
+  def index
+    schools = School.where(school_district_id: params[:school_district_id], school_type: params[:school_type]).pluck(:id, :name)
+    render json: ActiveModel::Serializer.new(schools, each_serializer: Api::V1::SchoolSerializer)
```

Comment:
`ActiveModel::Serializer.new` is implicit. This can be changed to:
```
render json: schools, each_serializer: Api::V1::SchoolSerializer
```

[example](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/api/v1/pd/workshops_controller.rb#L10)